lib/matplotlib/tests/test_category.py

```diff
@@ -160,9 +160,8 @@ def test_StrCategoryFormatter(self, ax, ydata):
     def test_StrCategoryFormatterPlot(self, ax, ydata, plotter):
         plotter(ax, range(len(ydata)), ydata)
         for i, d in enumerate(ydata):
-            assert ax.yaxis.major.formatter(i, i) == d
-        assert ax.yaxis.major.formatter(i+1, i+1) == ""
-        assert ax.yaxis.major.formatter(0, None) == ""
+            assert ax.yaxis.major.formatter(i) == d
+        assert ax.yaxis.major.formatter(i+1) == ""
```

Comment:
Could you add a test that `ax.yaxis.major.formatter(i, None)` gives `d` as the label? (which it wouldn't do before this fix)

Reply:
Thanks, I have updated the test.